src/clients/Fabric.js

```diff
@@ -1171,14 +1171,16 @@ const Fabric = {
     };
   },
 
-  CreateNonOwnerCap: async ({libraryId, objectId, publicKey, publicAddress, name}) => {
+  CreateNonOwnerCap: async ({libraryId, objectId, publicKey, label}) => {
     const {writeToken} = await Fabric.EditContentObject({libraryId, objectId});
+    publicKey = client.utils.HashToAddress(publicKey.replace("kupk", ""), true);
```

Comment:
let's inherently support both normal and mulltiformat public key. check if it starts with kupk. if so, decode it, otherwise assume it's already in the normal format.

easy way to avoid complaints / confusion if someone uses the normal public key format.